library/tasks/Hisat2.wdl

```diff
@@ -2,19 +2,19 @@
 ## run HISAT2 to genome reference with dedault parameters
 ## --seed to fix pseudo-random number and in order to produce deterministics results
 ## -k --secondary to output multiple mapping reads. --keep 10 will output up to 10 multiple mapping reads, which is default in HISAT2
-task HISAT2PE {
+task Hisat2PE {
   File hisat2_ref
   File fq1  # gz forward fastq file
   File fq2  # gz reverse fastq file
   String ref_name
   String output_name
   String sample_name
   Int disk_size
+
   command {
     # Note that files MUST be gzipped or the module will not function properly
     # This will be addressed in the future either by a change in how Hisat2 functions or a more
     # robust test for compression type.
-
     set -e
```

Comment:
Shall we have a section about mandating things like this for bash blocks with multiple commands? I didn't know about this and WDL/Cromwell debugging really depends on having this present, otherwise the error gets bumped to the next task.

Reply:
I purposely stayed away from the command block since that should be covered by bash best practices but if we feel that is important for us we can add it